docs/pricing/index.mdx

```diff
@@ -1,180 +1,163 @@
-| >100k-900k | $0.002565 | $0.003335 |
-| >900k-4.5M | $0.002309 | $0.003001 |
-| >4.5M-10M | $0.001962 | $0.002551 |
-
-#### Cron Monitors Pricing
+#### Tracing Pricing
 
-Monitors for Crons can't be purchased in advance. Crons are available to use and purchase through a predetermined on-demand budget that can be shared across all event types or defined per category.
+Tracing is enabled by spans and will be billed for in span units.
 
-| Total Monitors Volume | Team/Business On-Demand |
-|---|---|
-| >1 | $0.78 |
+| Span Unit Volume | Team Prepaid | Team PAYG* | Business Prepaid | Business PAYG* |
+| ---------------- | ------------ | ---------- | ---------------- | -------------- |
+| >10M-100M        | $0.0000016   | $0.0000020 | $0.0000032       | $0.0000040     |
+| >100M            | $0.0000014   | $0.0000018 | $0.0000029       | $0.0000036     |
 
-#### Attachments Pricing
 
-| Total Attachments Volume | Team/Business Prepaid | Team/Business On-Demand |
-|---|---|---|
-| >25 - 100 (GB) | $0.24 | $0.312 |
-| >100 - 200 (GB) | $0.24 | $0.312 |
-| >200 - 10k (GB) | $0.23 | $0.299 |
+#### Replays Pricing
 
-{/*<!-- prettier-ignore-end -->*/}
+| Replays Volume | Team Prepaid | Team PAYG* | Business Prepaid | Business PAYG* |
+| -------------- | ------------ | ---------- | -----------------| -------------- |
+| >50-5k         | $0.0030000   | $0.0037500 | $0.0030000       | $0.0037500     |
+| >5k-100k       | $0.0028500   | $0.0035630 | $0.0028500       | $0.0035630     |
+| >100k-900k     | $0.0025650   | $0.0032060 | $0.0025650       | $0.0032060     |
+| >900k-4.5M     | $0.0023090   | $0.0028860 | $0.0023090       | $0.0028860     |
+| >4.5M          | $0.0019620   | $0.0024530 | $0.0019620       | $0.0024530     |
 
-#### Consolidated Pricing
+#### Cron Monitors Pricing
 
-If the amount of prepaid data you want to buy falls in between increments, you may want to consider setting an on-demand budget instead, since on-demand is set in dollar amounts rather than data volume.
+Monitors for Crons can't be purchased in advance. Crons are available to use and purchase through a pay-as-you-go budget that can be shared across all event types.
 
-Use the slider on the [pricing](https://sentry.io/pricing/) page to determine the amount of prepaid data you might need based on the volume of event types you send.
+| Team PAYG* | Business PAYG* |
+| ---------- | -------------- |
+| $0.7800000 | $0.7800000     |
 
-![Reserved Volume settings in Subscriptions](./img/reserved-volume.png)
+#### Attachments Pricing
 
-For more information on how to manage your monthly spending, check out our [Quota Management documentation](/pricing/quotas/).
+| Attachments Volume | Team Prepaid | Team PAYG* | Business Prepaid | Business PAYG* |
+| ------------------ | ------------ | ---------- | ---------------- | -------------- |
+| >1G                | $0.2500000   | $0.3125000 | $0.2500000       | $0.3125000     |
 
-## Volume
+_** PAYG - pay-as-you-go.**_
```

Comment:
```suggestion
_* PAYG stands for pay-as-you-go._
```
I think this should be formatted as a footnote to explain what the * in the tables mean